core/runtime.rs

```diff
@@ -1425,23 +1413,22 @@ impl JsRuntime {
     let mut args: Vec<v8::Local<v8::Value>> =
       Vec::with_capacity(2 * overflown_responses_size);
```

Comment:
Please add a comment about the structure of the arguments sent back to JS

Reply:
Done